libcore/Widgets/BasicBreadcrumbsEntry.vala

```diff
@@ -421,12 +424,10 @@ namespace Files.View.Chrome {
             var l = (int)displayed_breadcrumbs.length (); //can assumed to be limited in length
             var w = displayed_breadcrumbs.first ().data.natural_width;
             if (l > 1) {
-                weak Gtk.StyleContext style_context = get_style_context ();
-                // var padding = style_context.get_padding (style_context.get_state ());
-                // w += (l - 1) * (MINIMUM_BREADCRUMB_WIDTH + padding.left + padding.right);
+                w += (l - 1) * (MINIMUM_BREADCRUMB_WIDTH);
```

Comment:
The commented-out code was summing two padding values. Is it okay that we’re no longer doing that?